kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -455,6 +460,18 @@ footer {
     width: 82px;
 }
 
+/* ===================================
+    Modal Dialog Login
+=================================== */
+
+.modal-dialog {
+    width: 400px;
+}
+
+div.panel-body form.form-horizontal .form-group {
```

Comment:
Ok, will work on more-specific-selector solution.
Question: if the changes are to be moved **from** bootstrap-overrides, where do we move them to?